src/ui/foundation/MainLayout/components/TopBar/TopBar.tsx

```diff
@@ -8,13 +8,15 @@ import {
 import { IconButton, Toolbar, Typography, useTheme } from '@mui/material';
 import { useNavigate } from 'react-router-dom';
 
-import { uiMain } from '@/ui/index';
 import { IpcWorkerMessageType } from '@/lib/ipc/ipcWorker';
 import { useSettingsLastActiveVaultId } from '@/ui/hooks/query';
+import { GlobalStateContext } from '@/ui/foundation/GlobalStateProvider';
+import { useContext } from 'react';
 
 export function TopBar() {
   const theme = useTheme();
   const navigate = useNavigate();
+  const { uiMain } = useContext(GlobalStateContext);
```

Comment:
Likely out of the scope of this PR, but are we going to move all calls to the backend to `hooks/queries`, so no component should have to use it directly?

Reply:
Yeah sure in follow-up PRs.